x-pack/plugins/lens/public/xy_visualization/xy_expression.tsx

```diff
@@ -332,7 +336,18 @@ export function XYChart({
         }
         legendPosition={legend.position}
         showLegendExtra={false}
-        theme={chartTheme}
+        theme={{
+          ...chartTheme,
+          ...(shouldEnrichThemeWithValueLabels && {
+            barSeriesStyle: {
```

Comment:
I think you might accidentally be overriding the rest of the barSeriesStyle options on the theme?

Also, I would prefer if we have a user-configurable font size. The default font size of 10 is too small and should be able to be increased in some cases.